src/scanner.ts

```diff
@@ -1266,57 +1242,4 @@ export class Scanner {
         return this.scanPunctuator();
     };
 
-    advance() {
```

Comment:
This is sweet.js's reader algorithm implementation and it's entirely moved to the Tokenizer class.